docs/source/daml-integration-kit/index.rst

```diff
@@ -96,35 +95,56 @@ support for running DAML.
    number of columns!
 
 .. list-table::
-   :widths: 25 25 25 25
+   :widths: 20 20 20 20 20
    :header-rows: 1
 
    * - Ledger
      - Status
-     - Developer
-     - More
+     - Integration
+     - Deployment
+     - More Information
    * - `VMware Blockchain <https://blogs.vmware.com/blockchain>`__
      - In Development
      - `VMware <https://www.vmware.com/>`__
-     - `press release (April 11th 2019)
+     - TBD
+     - `press release :: April 2019
        <http://hub.digitalasset.com/press-release/digital-asset-daml-smart-contract-language-now-extended-to-vmware-blockchain>`__
    * - `Hyperledger Sawtooth <https://sawtooth.hyperledger.org/>`__
      - In Development
      - `Blockchain Technology Partners <https://blockchaintp.com/>`__
-     - `press release (April 16th 2019)
+     - `Sextant <https://blockchaintp.com/sextant/>`__
+     - `press release :: April 2019
        <https://www.hyperledger.org/blog/2019/04/16/daml-smart-contracts-coming-to-hyperledger-sawtooth>`__
+   * - `Hyperledger Fabric <https://www.hyperledger.org/projects/fabric>`__
+     - In Development
+     - `Hacera <https://hacera.com>`__     
+     - `Hacera Unbounded Network <https://unbounded.network/>`__
+     - `press release :: June 2019
+       <https://hub.digitalasset.com/press-release/digital-asset-announces-daml-partner-integrations-with-hyperledger-fabric-r3-corda-and-amazon-aurora>`__
+   * - `R3 Corda <https://www.corda.net>`__
+     - In Development
+     - `Digital Asset <https://digitalasset.com/>`__
+     - TBD
+     - `press release :: June 2019
+       <https://hub.digitalasset.com/press-release/digital-asset-announces-daml-partner-integrations-with-hyperledger-fabric-r3-corda-and-amazon-aurora>`__
+   * - `Amazon Aurora <https://www.hyperledger.org/projects/fabric>`__
+     - In Development
+     - `Blockchain Technology Partners <https://blockchaintp.com/>`__
+     - `Sextant <https://blockchaintp.com/sextant/>`__     
+     - `press release :: June 2019
+       <https://hub.digitalasset.com/press-release/digital-asset-announces-daml-partner-integrations-with-hyperledger-fabric-r3-corda-and-amazon-aurora>`__
    * - :doc:`/tools/sandbox`
      - Stable
      - `Digital Asset <https://digitalasset.com/>`__
+     - TBD
      - `PostgreSQL <https://www.postgresql.org/>`__ backend coming soon
```

Comment:
This is actually done, right? We could link to https://docs.daml.com/tools/sandbox.html#running-with-persistence.